client.go

```diff
@@ -424,6 +430,18 @@ func New(cfg Config) (c Client, err error) {
 		}
 	}
 
+	if cfg.RequestObjectEncryptionKeyFile != "" {
+		var bs []byte
+
+		if bs, err = ioutil.ReadFile(cfg.RequestObjectEncryptionKeyFile); err != nil {
```

Comment:
According to gocritic,  ioutil.Readfile has been deprecated in favor of os.ReadFile.

Reply:
Ah, you're right

Reply:
Fixed